src/txn/txn_timestamp.c

```diff
@@ -432,17 +434,25 @@ __wt_txn_global_set_timestamp(WT_SESSION_IMPL *session, const char *cfg[])
 	if (has_commit && (has_oldest || txn_global->has_oldest_timestamp) &&
 	    __wt_timestamp_cmp(&oldest_ts, &commit_ts) > 0) {
 		__wt_readunlock(session, &txn_global->rwlock);
+		WT_RET(__wt_timestamp_to_hex_string(
+		    session, hex_timestamp1, &oldest_ts));
+		WT_RET(__wt_timestamp_to_hex_string(
+		    session, hex_timestamp2, &commit_ts));
 		WT_RET_MSG(session, EINVAL,
-		    "set_timestamp: oldest timestamp must not be later than "
-		    "commit timestamp");
+		    "set_timestamp: oldest timestamp %s must not be later than"
```

Comment:
I think the convention is to put the space at the end of the previous line and not start the next line with a space. Same at the other places.